nautobot/dcim/api/serializers.py

```diff
@@ -759,22 +813,39 @@ def validate(self, data):
         return super().validate(data)
 
 
-class RearPortSerializer(NautobotModelSerializer, TaggedModelSerializerMixin, CableTerminationModelSerializerMixin):
+class RearPortSerializer(
+    ModularDeviceComponentSerializerMixin,
+    NautobotModelSerializer,
+    TaggedModelSerializerMixin,
+    CableTerminationModelSerializerMixin,
+):
```

Comment:
While you're touching all of these anyway, would it make sense to fix the MRO for the other mixins (i.e. make `NautobotModelSerializer` the last in all cases)?